matrix-appservice-slack/docker-compose.postgres.yml

```diff
@@ -0,0 +1,21 @@
+
+version: '3.1'
+
+services:
+  postgres:
+    image: postgres:11
+    environment:
+      POSTGRES_PASSWORD: postgres
+    command: -c fsync=off
+
+  testenv:
+    image: node:14
+    depends_on:
+      - postgres
+    environment:
+      SLACKBRIDGE_TEST_PGDB: "slackintegtest"
+      SLACKBRIDGE_TEST_PGURL: "postgresql://postgres:postgres@postgres"
+    working_dir: /app
+    volumes:
+      - ..:/app
```

Comment:
what is this `..` relative to? mounting the parent of the build directory sounds like a dangerous idea.

Reply:
Hm, I had changed that. Anyway, commited the correct mapping.